server/boot/randomAPIs.js

```diff
@@ -171,26 +171,27 @@ module.exports = function(app) {
 
   function unsubscribeAll(req, res, next) {
     req.checkParams('email', 'Must send a valid email').isEmail();
-    return User.findOne({ where: { email: req.params.email } }, (err, user) => {
+    var query = { email: req.params.email };
+    var params = {
+      sendQuincyEmail: false,
+      sendMonthlyEmail: false,
+      sendNotificationEmail: false
+    };
+    var options = { multi: true };
+    return User.update(query, params, options, (err, updates) => {
```

Comment:
I've updated the function call to use updateAll():
```
  function unsubscribeAll(req, res, next) {
    req.checkParams('email', 'Must send a valid email').isEmail();
    var query = { email: req.params.email };
    var params = {
      sendQuincyEmail: false,
      sendMonthlyEmail: false,
      sendNotificationEmail: false
    };
    return User.updateAll(query, params, function(err, info) {
      console.log(err);
      if (err) { return next(err); }
      console.log(info);
      if (info.count === 0) {
        req.flash('info', {
          msg: 'Email address not found. ' +
          'Please update your Email preferences from your profile.'
        });
        return res.redirect('/map');
      } else {
        req.flash('info', {
          msg: 'We\'ve successfully updated your Email preferences.'
        });
        return res.redirect('/unsubscribed');
      }
    });

```

The issue is that `info.count` returns a number larger than 0, but the function call does not update the mongo database. The attributes:

```
"sendMonthlyEmail" : true, 
"sendNotificationEmail" : true,
"sendQuincyEmail" : true
```
remain `true` even after the function call has been made. The weird thing is that the error value is `null` and `info.count` is greater than 0 so the application assumes the change has been made, but when I query the users using the mongo client I can see that it hasn't changed. There are no obvious errors in the console, so I am not sure what's wrong. Any ideas?